README.md

```diff
@@ -1,35 +1,99 @@
+* [Configuring the Arlec Smart Plug](https://github.com/WebThingsIO/wiki/wiki/Arlec-Smart-Plug)
+* [Factory reset a Cree Connected bulb](https://github.com/WebThingsIO/wiki/wiki/HOWTO%3A-Factory-reset-a-Cree-Connected-bulb)
+* [Factory reset a Hue bulb](https://github.com/WebThingsIO/wiki/wiki/HOWTO%3A-Factory-reset-a-Hue-bulb)
+* [Factory reset a Hue Wireless Dimmer](https://github.com/WebThingsIO/wiki/wiki/HOWTO%3A-Factory-reset-a-Hue-Wireless-Dimmer)
+* [Factory reset an IKEA bulb](https://github.com/WebThingsIO/wiki/wiki/HOWTO%3A-Factory-reset-an-IKEA-bulb)
 
-### Development
-* 🌐 [Docker Image](https://hub.docker.com/r/mozillaiot/gateway)
-* 🌐 [Build Instructions](https://github.com/WebThingsIO/gateway#readme)
 
-## WebThings Framework
+## Developer Guide
 
-[WebThings Framework](https://iot.mozilla.org/framework/) is a collection of re-usable software components to help developers build their own web things which directly expose the Web Thing API.
+### Web Thing API
+The Web Thing API is the REST & WebSockets API used by the WebThings IoT platform for monitoring and controlling devices over the web. Parts of the Web Thing API specification are currently being standardised via the W3C.
+* [Introduction to the Web Thing API](https://github.com/WebThingsIO/wot/blob/gh-pages/README.md)
+* [Web Thing API specification](https://webthings.io/api/)
+  * [Examples using curl](https://github.com/WebThingsIO/curl-examples/blob/master/README.md)
+  * [Differences with W3C specification](https://github.com/webthingsio/wiki/wiki/Mozilla-W3C-Differences)
+* [WoT Capability Schemas](https://webthings.io/schemas/)
 
-### Libraries
-* 🌐 [Node.js](https://github.com/WebThingsIO/webthing-node)
-* 🌐 [Python](https://github.com/WebThingsIO/webthing-python)
-* 🌐 [Java](https://github.com/WebThingsIO/webthing-java)
-* 🌐 [Rust](https://github.com/WebThingsIO/webthing-rust)
-* 🌐 [Arduino](https://github.com/WebThingsIO/webthing-arduino)
-* 🌐 [MicroPython](https://github.com/WebThingsIO/webthing-upy)
+### WebThings Gateway
+[WebThings Gateway](https://webthings.io/gateway/) is an open source implementation of a Web of Things gateway which bridges a range of different IoT protocols to the Web Thing API and provides a web interface for users to monitor and control devices.
+* [Gateway Architecture](https://github.com/WebThingsIO/wiki/wiki/Gateway-Architecture)
+* [Build Instructions](https://github.com/WebThingsIO/gateway#readme)
+* [Creating a new translation](https://github.com/WebThingsIO/wiki/wiki/Fluent%3A-Making-a-new-translation)
+* [Testing pre-release OTA updates](https://github.com/WebThingsIO/wiki/wiki/Testing-prerelease-OTA-updates)
+* [Releasing a Gateway OTA Update](https://github.com/WebThingsIO/wiki/wiki/How-To-Release-a-Gateway-OTA-Update)
+* [Using the staging server](https://github.com/WebThingsIO/wiki/wiki/HOWTO%3A-Use-the-staging-server-for-testing)
```

Comment:
I'd suggest removing this staging server stuff... it should probably be removed from the wiki as well.

Reply:
Yeah I wondered about that. There's quite a lot of documentation on the wiki that's out of date actually.